frontend/src/components/UploadArea/UploadArea.tsx

```diff
@@ -12,6 +12,20 @@ const UploadArea: FunctionComponent<UploadingAreaProps> = ({ value, onChange }):
   const uploadInputRef = useRef<HTMLInputElement | null>(null);
   const setParsedFile = useSetRecoilState(excelFileAtom);
 
+  const handleFile = async (file: File) => {
+    onChange(file);
+    setParsedFile({ state: "uploading", progress: 0 });
+    try {
+      const targets = await Api.parseExcelFile(file, () => {});
```

Comment:
oh, it's my bad. this callback shouldn't be empty. there should be updating `p => setParsedFile({ state: "uploading", progress: p });`